### DISCUSSION TITLE
Thanks for the length checks

### DISCUSSION THREAD

**Discussion by danielfdickinson** [bowman2001/perplexdoc#27] [General]
As the title says. It will take me a while to work through these, but I think it is worth it.
I made the mistake of 'fixing' many of my posts based on the Ahrefs (which came from Bing AFAICT) suggested lengths. I think the one you have picked are more in line with Google's, not to mention looking better.

**Comment by bowman2001** [comment]
You’re welcome. FYI: You find the new parameters in the theme config, they aren’t documented yet. You can change the limits and suppress warnings separately which should help with focusing on one parameter at a time.

**Comment by danielfdickinson** [comment]
Works like a charm. I'm working through the Princes and Madmen site first, but plan on getting everything ship-shape.
